src/parsers/native_format_parsers.js

```diff
@@ -1,10 +1,11 @@
+import Espresso from "./espresso";
 import Poscar from "./poscar";
 
 const NATIVE_FORMAT = {
     JSON: "json",
     POSCAR: "poscar",
     CIF: "cif",
-    PWX: "pwx",
+    IN: "in",
```

Comment:
Probably better to call this `QE`, maybe?

Reply:
I thought in terms of file extension. .json, .cif, .in